db/migrate/20211206132802_rename_author_column_in_comments.rb

```diff
@@ -0,0 +1,5 @@
+class RenameAuthorColumnInComments < ActiveRecord::Migration[6.1]
+    def change
+      rename_column :comments, :author_id, :user_id
+    end
```

Comment:
Please notice the error when running r `rails db:migrate` and fix it so the user can install the project without any errors.
![migrate](https://user-images.githubusercontent.com/35504149/145062146-983eaed3-0967-4898-b7cc-1fef1d670e98.png)